develop/dev-guide-sample-application-java-jdbc.md

```diff
@@ -74,7 +74,7 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 
 3. Ensure the configurations in the connection dialog match your operating environment.
 
-    - **Endpoint Type** is set to `Public`
+    - **Connection Type** is set to `Public`
```

Comment:
https://github.com/pingcap/docs/pull/18756 renames **Endpoint Type** to **Connection Type** for TiDB Serverless clusters